sccm/sum/deploy-use/third-party-software-updates.md

```diff
@@ -31,18 +31,18 @@ Beginning with version 1806, the **Third-Party Software Update Catalogs** node i
 
 ## Additional requirements when the SUP is remote from the top-level site server 
 
-1. SSL must be enabled on the SUP when it's remote. 
+1. SSL must be enabled on the SUP when it's remote. This requires a server authentication certificate generated from an internal certififcate authority or via a public provider.
     - [Configure SSL on WSUS](https://docs.microsoft.com/windows-server/administration/windows-server-update-services/deploy/2-configure-wsus#bkmk_2.5.ConfigSSL)
         - When you configure SSL on WSUS, note some of the web services and the virtual directories are always HTTP and not HTTPS. 
         - Configuration Manager downloads third-party content for software update packages from your WSUS content directory over HTTP.   
     - [Configure SSL on the SUP](../get-started/install-a-software-update-point.md#configure-ssl-communications-to-wsus)
 
-2. To allow the creation of the self-signed WSUS certificate: 
+2. When setting third-party updates WSUS signing certificate configuration to **Configuration Manager manages the updates** in the Software Update Point Component Properties, the following is required to allow the creation of the self-signed WSUS certificate: 
    - Remote registry should be enabled on the SUP server.
    -  The **WSUS server connection account** should have remote registry permissions on the SUP/WSUS server. 
 
 
-3. Create the following registry key on the Configuration Manager site server: 
+3. When using a self-signed certificate, create the following registry key on the Configuration Manager site server: 
```

Comment:
We don't want to make this configuration conditional right now. We've only tested with this setting enabled. So perhaps leave it as-is?

Reply:
This is the only bit about which I still have reservations. Let's keep it an explicit action for all configurations, as it was originally, "Create the following registry key...." Otherwise it leaves room for interpretation and potential confusion.

Reply:
OK. I'm only basing this on the DWORD EnableSelfSignedCertificates. Otherwise can that edit.